changelogs/fragments/677-jenkins_plugins_sha1.yaml

```diff
@@ -0,0 +1,5 @@
+---
+bugfixes:
+  - jenkins_plugins - replace MD5 checksum verification with SHA1 due to MD5
```

Comment:
```suggestion
  - jenkins_plugin - replace MD5 checksum verification with SHA1 due to MD5
```

Reply:
Updated in most recent commit. Thanks for catching this one :-X

Reply:
No problem! BTW, I don't think you pushed this commit yet (the changelog file didn't change in the last commit).

Reply:
Pushed now, thanks!